bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -731,6 +733,14 @@ public Journal(int journalIndex, File journalDirectory, ServerConfiguration conf
             LOG.debug("Last Log Mark : {}", lastLogMark.getCurMark());
         }
 
+        try {
+            this.fileChannelProvider = FileChannelProvider.newProvider(conf.getJournalChannelProvider());
+        } catch (IOException e) {
+            LOG.warn("Failed to initiate {}, use default fileSystem provider instead.",
+                conf.getJournalChannelProvider());
+            this.fileChannelProvider = new DefaultFileChannelProvider();
```

Comment:
We should fail the bookie, otherwise the system will not work as configured by the administrator

Reply:
done